apps/api/src/app/environments-v1/novu-bridge-client.ts

```diff
@@ -46,12 +45,13 @@ export class NovuBridgeClient {
         ConstructFrameworkWorkflowCommand.create({
           environmentId: req.params.environmentId,
           workflowId: req.query.workflowId as string,
+          controlValues: req.body.controls,
+          stepId: req.body.stepId,
```

Comment:
☑ chore: These parameters are no longer needed after the removal of transient previews. Please remove them.

Reply:
controlValues is needed, just the step id isn't

Reply:
I can't see that we access `command.controlValues` in the `ConstructFrameworkWorkflow` use-case, could you please point me to it?

Reply:
it's a bug, thank you for catching

Reply:
it will be a part of constructing the step